config/application.rb

```diff
@@ -190,6 +190,14 @@ class Application < Rails::Application
     config.logger = Foreman::Logging.logger('app')
     config.active_record.logger = Foreman::Logging.logger('sql')
 
+    if config.serve_static_assets
```

Comment:
I don't think this setting is being respected at all, `application.rb` runs before `environment/****.rb` ?